### PR TITLE
[MM-29336] Add e2e tests to channelUsersAddCmdF

### PR DESCRIPTION
**Pull request by simross**
**Summary**

Adds e2e tests for channelUsersAddCmdF.

**Ticket Link**

Fixes mattermost/mattermost-server#15814

JIRA: https://mattermost.atlassian.net/browse/MM-29336

### CODE REVIEW

File: commands/channel_users_e2e_test.go [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+	}, false)
+	s.Require().Nil(appErr)
+
+	s.RunForSystemAdminAndLocal("Add user to nonexistent channel", func(c client.Client) {
+		printer.Clean()
+
+		nonexistentChannelName := "nonexistent"
+		err := channelUsersAddCmdF(c, &cobra.Command{}, []string{nonexistentChannelName, user.Id})
+		s.Require().NotNil(err)
+		s.Require().Equal(fmt.Sprintf("unable to find channel \"%s\"", nonexistentChannelName), err.Error())
```
Comment (agnivade) [inline commands/channel_users_e2e_test.go:42]: We can use `%q`

File: commands/channel_users_e2e_test.go [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+		s.Require().Nil(appErr)
+		defer func() {
+			appErr := s.th.App.RemoveUserFromChannel(s.th.BasicUser.Id, s.th.SystemAdminUser.Id, channel)
+			s.Require().Nil(appErr)
+		}()
+
+		nonexistentChannelName := "nonexistent"
+		err := channelUsersAddCmdF(s.th.Client, &cobra.Command{}, []string{nonexistentChannelName, user.Id})
+		s.Require().NotNil(err)
+		s.Require().Equal(fmt.Sprintf("unable to find channel \"%s\"", nonexistentChannelName), err.Error())
```
Comment (agnivade) [inline commands/channel_users_e2e_test.go:60]: Same as above

File: commands/channel_users_e2e_test.go [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+	s.Require().Nil(appErr)
+
+	_, appErr = s.th.App.AddUserToTeam(s.th.BasicTeam.Id, user.Id, "")
+	s.Require().Nil(appErr)
+
+	channelName := api4.GenerateTestChannelName()
+	channel, appErr := s.th.App.CreateChannel(&model.Channel{
+		TeamId:      s.th.BasicTeam.Id,
+		Name:        channelName,
+		DisplayName: "db_" + channelName,
```
Comment (agnivade) [inline commands/channel_users_e2e_test.go:31]: What does `db_` mean here?

Comment (simross) [reply commands/channel_users_e2e_test.go]: Typo, it was meant to be "dn" - short for display name. This approach is used in mattermost-server tests. Fixed.

File: commands/channel_users_e2e_test.go (hunk 2 above)
Comment (simross) [reply commands/channel_users_e2e_test.go]: Fixed.

File: commands/channel_users_e2e_test.go (hunk 1 above)
Comment (simross) [reply commands/channel_users_e2e_test.go]: Fixed.

**Comment by agnivade** [comment]
Looks good, but there seems to be some unrelated commits which went in. I think you might need to remove those.

**Comment by simross** [comment]
> Looks good, but there seems to be some unrelated commits which went in. I think you might need to remove those.

My bad. Reverted unrelated commits.

**Review by agnivade — APPROVED** [review]
Looks great!

File: commands/channel_users_e2e_test.go [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+	"github.com/spf13/cobra"
+
+	"github.com/mattermost/mmctl/client"
+	"github.com/mattermost/mmctl/printer"
+)
+
+func (s *MmctlE2ETestSuite) TestChannelUsersAddCmdF() {
+	s.SetupTestHelper().InitBasic()
+
+	s.th.CreateUser()
```
Comment (mgdelacroix) [inline commands/channel_users_e2e_test.go:20]: This line can be removed

**Review by mgdelacroix — APPROVED** [review]
LGTM! :rocket: Thanks @simross!!

Added a nonblocking comment to remove a line

**Review by lindy65 — APPROVED** [review]
Thanks @simross @mgdelacroix - looks good to me 👍

**Pull request merged**